tests/qir_utils.py

```diff
@@ -324,6 +324,8 @@ def check_single_qubit_rotation_op(
         return
     for line in entry_body:
         if line.strip().startswith("call") and f"qis__{gate_name}" in line:
+            print("actual:", line.strip())
+            print("expected:", rotation_call_string(gate_name, param_list[q_id], qubit_list[q_id]))
```

Comment:
Totally fine for debugging but don't want to permanently include print statements in the tests so either want to delete or move to the comment following the assert statement below